budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/service/user/UserService.java

```diff
@@ -118,4 +130,14 @@ public void registerUser(String username, String password) throws UsernameAlread
             throw new UsernameAlreadyInUseException();
         }
     }
+
+    /**
+     * Register user without password if using Keycloak
+     */
+    public void registerUser(String username) {
```

Comment:
as this method is only used in this class this method should be private or at least package private.